tests/Unit/ValueObjects/PropertyMapTest.php

```diff
@@ -62,7 +70,9 @@ public function testCanBeConvertedToJson(): void
 
         self::assertIsString($mapAsJson);
         self::assertJsonStringEqualsJsonString(
+            // phpcs:disable Generic.Files.LineLength
```

Comment:
I would keep the PHPCS annotations out of the PR. Just not a too big fan of it.
In my opinion there is no issue with having some long lines in the code (if logical for that specific LOC) esp. in the `tests` folder.

Reply:
Me neither, but i want clear that issues, this was most specific solution available.
So do you prefer have some issues here (current state of code), or skip that line length for entire test file with custom phpcs.xml.dist file?

Reply:
The custom `phpcs.xml.dist` sounds like a reasonable suggestion. This way we keep the PHPCS specific things into its config file. If we can add the skipping of line length for just the `tests` folder that would be best. Would you be able to find out how this can be done?

Reply:
I had it prepared. But I only exclude files with problems, not entire tests folder. I think it's better for cleaner code - line length issues in other tests are checked and must by fixed, file will be removed from checking only in exceptional cases, where it make sense (so far they are: data provider definition and check for JSON string).